Library/Homebrew/extend/os/linux/linkage_checker.rb

```diff
@@ -41,13 +41,11 @@ def display_deprecated_warning(strict: false)
     # If there's no library deprecated/disabled handling left:
     # - Remove the `display_` overrides here and the associated generic aliases in HOMEBREW_LIBRARY/linkage_checker.rb
 
-    odisabled "linkage to libcrypt.so.1", "libcrypt.so.2 in the libxcrypt formula" if @libcrypt_found
-
     return unless @libnsl_found
 
-    odeprecated "linkage to libnsl.so.1", "libnsl.so.3 in the libnsl formula",
-                disable:                fail_on_libnsl1?(strict: strict),
-                disable_for_developers: false
+    odisabled "linkage to libnsl.so.1", "libnsl.so.3 in the libnsl formula",
+              disable:                fail_on_libnsl1?(strict: strict),
+              disable_for_developers: false
```

Comment:
As per the instructions a few lines up:

```suggestion
    odisabled "linkage to libnsl.so.1", "libnsl.so.3 in the libnsl formula"
```

Also: remove from `SYSTEM_LIBRARY_ALLOWLIST`, remove `fail_on_libnsl1?` and the associated checks in `broken_library_linkage?`.

Depending on when you are planning to merge this, it may be worth setting `HOMEBREW_DISALLOW_LIBNSL1` in homebrew-core CI until this is merged if we want to test this in advance for enabling it globally (though libnsl1 is much rarer than libcrypto1).

Reply:
@Bo98 Sorry, I did read this instructions and attempted to follow them. Will update this.

> Depending on when you are planning to merge this,

Probably in about a week and have it bake for a week?